Unix/doc/setup-ntlm-omi.md

```diff
@@ -177,9 +177,92 @@ On the command line:
 
 ##Enabling SPNEGO/NTLM on the OMI server
 
+Since it is the most secure, by default, only NTLM authentication via winbind is configured in omiserver. There may be reasons
+why this would not be desirable for a particular implementation, in which case it is possible to define a credential file,
+using the same format as described for the client. 
 
-The most secure method for providing omiserver credentials is via winbind. There may be reasons why this would not be 
-practical for a particular implementation, in which case it is possible to define a credential file, using the same format as
-described for the client.  If this method is used, the location of the file is specified in the omiserver.conf file.
+If this method is used, use of the file is specified by setting the option 
+```
+NtlmCredsFile= *path* 
+```
+in the configuration file omiserver.conf.  The file then refered to by the server for credentials. Like certificate directories,
+the server requires permissions of 700 for the directory and 600 for the file. Both the file and directory must be owned by root.
+If the correct permissions and ownership are not set the file will be ignored with an error in the server log.  
+
+The creds/ntlm file must contain credentials of each authorised user with each domain (including hostname). IP addresses 
+in the NTLM domain are treated as different from hostname, so a separate entry is required for each.
+
+
+## Troubleshooting Authentication
+
+
+###The Process:
+
+-the omi client requests authentication using the username and optional password to acquire a stored credential.
```

Comment:
These aren't displaying properly in markdown, likely because of lack of space after the dash. Please fix and view the file in the PR before posting again. I'd like folks to review their own markdown themselves prior to posting a PR since it's easy to make a mistake in this area.